charts/gardener/controlplane/charts/runtime/templates/admission-controller/configmap-componentconfig.yaml

```diff
@@ -15,6 +15,8 @@ data:
     ---
     apiVersion: admissioncontroller.config.gardener.cloud/v1alpha1
     kind: AdmissionControllerConfiguration
+    logLevel: {{ .Values.global.admission.config.logLevel | default "info" }}
+    logFormat: {{ .Values.global.admission.config.logFormat | default "json" }}
```

Comment:
Nit: We could follow the fields order in the types.go. Currently it is a little bit confusing as log fields are put at the beginning in this template file, in the example file - they are added almost at the end of the file.

```diff
@@ -15,8 +15,6 @@ data:
     ---
     apiVersion: admissioncontroller.config.gardener.cloud/v1alpha1
     kind: AdmissionControllerConfiguration
-    logLevel: {{ .Values.global.admission.config.logLevel | default "info" }}
-    logFormat: {{ .Values.global.admission.config.logFormat | default "json" }}
     gardenClientConnection:
       {{- with .Values.global.admission.config.gardenClientConnection.acceptContentTypes }}
       acceptContentTypes: {{ . | quote }}
@@ -32,6 +30,8 @@ data:
       {{- if .Values.global.admission.kubeconfig }}
       kubeconfig: /etc/gardener-admission-controller/kubeconfig/kubeconfig
       {{- end }}
+    logLevel: {{ .Values.global.admission.config.logLevel | default "info" }}
+    logFormat: {{ .Values.global.admission.config.logFormat | default "json" }}
     server:
       https:
         bindAddress: {{ required ".Values.global.admission.config.server.https.bindAddress is required" .Values.global.admission.config.server.https.bindAddress }}
```